packages/contracts-periphery/contracts/universal/citizen-house/BadgeAdmin.sol

```diff
@@ -0,0 +1,622 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { IERC721 } from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import { IBadge } from "./IBadge.sol";
```

Comment:
To be honest I don't think we need an `IBadge` interface, we could just import `Badge` and use that to cast addresses to get the nice API when calling them